framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1478,8 +1478,10 @@ public void preProcessClassTree(ClassTree classTree) {}
      * to override this method if storing defaulted types is not desirable.
      */
```

Comment:
@nicholaschew11 Hi Nicholas, can you try to add javadoc for classtree parameter just like https://github.com/eisop/checker-framework/blob/7f542623a5413eb6d0b13893961c1020e9b8860c/framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java#L1468? This will avoid misc check fail as you can see from https://dev.azure.com/eisop/checker-framework/_build/results?buildId=3861&view=logs&jobId=03ef1a5a-d30a-57af-3a6d-62ddb34c1ad2&j=03ef1a5a-d30a-57af-3a6d-62ddb34c1ad2&t=5d54b24c-372d-5954-02b2-e775bd9d239f